### ISSUE TITLE
Consider moving `watertap/tools` to a separate repository

### ISSUE THREAD

**Issue by bknueven** [watertap-org/watertap#1107]
- Capability is orthogonal to core WaterTAP
- However, our examples to utilize the capabilities `watertap/tools`

**Comment by avdudchenko** [comment]
I agree, this is something I suggested before, as arguably, the tools developed here in are not necessarily water tap specific either (IDAES could benefit from them as well).

We will also see the tools code base grow with further mergers from analysis tool kit, there is a datamangement tool, figure generation tool, and later I hope an even higher API tool that automates data management and fig generation for data genrated from PS tool/loopTool. (data collator tool). 

(For those curios, you can check the rough code for these here -  https://github.com/watertap-org/NAWI-analysis/tree/main/analysis_generic_modules/plot_kit/analysis_plot_kit/core ) 

This might also be a good place to move things like OLI api (@adam-a-a) and phreeqc APIs .
